src/modules/service-catalog/ServiceCatalog.tsx

```diff
@@ -1,5 +1,39 @@
-import { Span } from "@zendeskgarden/react-typography";
+import { Row, Col } from "@zendeskgarden/react-grid";
+import { OrderedList } from "@zendeskgarden/react-typography";
+
+import { useCallback, useEffect, useState } from "react";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
 
 export function ServiceCatalog() {
-  return <Span>Hello from module</Span>;
+  const [serviceCatalogItems, setServiceCatalogItems] = useState([]);
+  const fetchServiceCatalogItems = useCallback(async () => {
+    try {
+      const response = await fetch(
+        `/api/v2/custom_objects/service_catalog_item/records`
+      );
+      if (response.ok) {
+        const { custom_object_records } = await response.json();
+        setServiceCatalogItems(custom_object_records);
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  useEffect(() => {
+    fetchServiceCatalogItems();
+  }, []); //for now it will be rendered only once
+
+  return (
+    <Row>
+      <Col>
+        <OrderedList>
+          {serviceCatalogItems?.length !== 0 &&
```

Comment:
Do we need the optional mark `?` on `serviceCatalogItems`?
Seems like the default is an empty so it shouldn't be necessary (?)

Reply:
yes, we don't need that, I removed it.